### ISSUE TITLE
UnicodeDecodeError when saltutil.sync_modules for Windows  salt-minion  2015.8.3

### ISSUE THREAD

**Issue by WangWenchao** [saltstack/salt#29918]
UnicodeDecodeError when saltutil.sync_modules for Windows  salt-minion  2015.8.3

```
salt '<minion_id>' saltutil.sync_modules
<minion_id>:
    The minion function caused an exception: Traceback (most recent call last):
      File "C:\salt\bin\lib\site-packages\salt\minion.py", line 1039, in _thread_return
        return_data = func(*args, **kwargs)
      File "C:\salt\bin\lib\site-packages\salt\modules\saltutil.py", line 331, in sync_modules
        ret = _sync('modules', saltenv)
      File "C:\salt\bin\lib\site-packages\salt\modules\saltutil.py", line 112, in _sync
        source, sub_env, include_pat=r'E@\.(pyx?|so|zip)$'
      File "C:\salt\bin\lib\site-packages\salt\modules\cp.py", line 444, in cache_dir
        path, saltenv, include_empty, include_pat, exclude_pat
      File "C:\salt\bin\lib\site-packages\salt\fileclient.py", line 233, in cache_dir
        if fn_.strip() and fn_.startswith(path):
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 85: ordinal not in range(128)

```

**Comment by ssgward** [comment]
@WangWenchao - thanks for the report

**Comment by jfindlay** [comment]
@WangWenchao, do you know if this was not happening with previous versions of salt?

**Comment by WangWenchao** [comment]
@jfindlay thank your , this was not happened with previous version of salt ,  as I update salt from 2014.7  to 2015.5 ,  now to 2015.8.1   during 2015 ,  but when we upgrade to 2015.8.3 I get the problem ,  have back to 2015.8.1  .

**Comment by gaoyang-gridsum** [comment]
i found the same error at 2015.8.3 ,and  is not happened with 2015.5.0

**Comment by jfindlay** [comment]
@gaoyang-gridsum, this should be fixed in 2015.8.4.
